common/lib/protocol-definitions/src/consensus.ts

```diff
@@ -54,6 +60,8 @@ export interface IQuorumProposalsEvents extends IErrorEvent {
         listener: (
             sequenceNumber: number,
             key: string,
+            // TODO: use `unknown` instead.
```

Comment:
Do you intend to do this in this PR? Might be risky to do in a minor version.

Reply:
No, not in this PR. Should be done at some point though. I am happy to do a follow-up in next to migrate some of these things after I've finished the linter dep update.